app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -105,6 +105,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="@dimen/add_profile_pin_checkbox_margin_top"
+            android:textColor="@color/add_profile_activity_label_text_color"
```

Comment:
Why was this attribute added, as earlier it was not available?

Reply:
That is needed because otherwise the text along with the check-box stays black even in dark-mode.

Reply:
Ohh okay got it.